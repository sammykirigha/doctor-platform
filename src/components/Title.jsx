import React from "react";

const Title = ({ title }) => {
    return (
        <div>
            <h2 className="text-gray-600 text-xl font-semibold">{title}</h2>
        </div>
    );
};

export default Title;
