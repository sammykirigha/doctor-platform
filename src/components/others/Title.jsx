import React from "react";

const Title = ({ title }) => {
    return (
        <div className="mt-0">
            <h2 className="text-gray-600 text-xl font-semibold">{title}</h2>
        </div>
    );
};

export default Title;
