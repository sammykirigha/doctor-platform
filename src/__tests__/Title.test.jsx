import { cleanup, render, screen } from '@testing-library/react';
import Title from '../components/Title'


describe('The suit is to test the body component', () => {

	it('Snapshot of UserMessage', () => {
		const { asFragment } = render(<Title title="my header" />)
		expect(asFragment()).toMatchSnapshot()
	})

	it('should render the UserMessage with the right index', () => {
		render(<Title title="my header" />)
		let element = screen.getByText('my header');

		expect(element.textContent).toBe('my header')
	})

})