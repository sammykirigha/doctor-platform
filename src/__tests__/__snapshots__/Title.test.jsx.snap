// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The suit is to test the body component Snapshot of UserMessage 1`] = `
<DocumentFragment>
  <div>
    <h2
      class="text-gray-600 text-xl font-semibold"
    >
      my header
    </h2>
  </div>
</DocumentFragment>
`;
